doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -86,32 +89,32 @@ Steps for Using Zephyr as User VM
 
    .. code-block:: none
 
-      $ sudo umount /mnt
+      sudo umount /mnt
 
-   You now have a virtual disk image with a bootable Zephyr in ``zephyr.img``. If the Zephyr build system is not
+   You now have a virtual disk image with a bootable Zephyr in ``zephyr.img``.
+   If the Zephyr build system is not
    the ACRN Service VM, then you will need to transfer this image to the
-   ACRN Service VM (via, e.g, a USB drive or network)
-
-#. Follow :ref:`gsg`
-   to boot "The ACRN Service OS" based on Ubnuntu OS (ACRN tag: v2.2)
+   ACRN Service VM (via, e.g., a USB drive or network).
 
+#. Follow :ref:`gsg` to boot the Service VM based on Ubuntu OS (ACRN tag: v2.2)
```

Comment:
Our (current) GSG doesn't say to use ACRN v2.2 , so I (hope) we can simply remove this parenthetical phrase.